core/common/src/main/java/alluxio/retry/RetryUtils.java

```diff
@@ -99,6 +99,13 @@ public static RetryPolicy defaultWorkerMasterClientRetry() {
         .build();
   }
 
+  /**
+   * @return one shot retry policy that just try once
+   */
+  public static RetryPolicy oneshotRetryPolicy() {
+    return new CountingRetry(1);
+  }
```

Comment:
I wonder if there is an existing `RetryPolicy` that just does no retry?

Reply:
you can input CountingRetry(0) for this purpose